conda_forge_webservices/feedstock_outputs.py

```diff
@@ -140,6 +140,7 @@ def copy_feedstock_outputs(outputs, channel, delete=True):
                     to_owner=PROD,
                     from_label=channel,
                     to_label=channel,
+                    replace=True,
```

Comment:
Where are the docs on this?

Reply:
This will overwrite files too. Normally we don't get duplicate uploads, but if we happen to, we'd nuke our own files. That is too dangerous to use in production

Reply:
https://github.com/Anaconda-Platform/anaconda-client/blob/master/binstar_client/mixins/package.py#L35

Reply:
It's visible via the CLI. Also here's [the source code]( https://github.com/Anaconda-Platform/anaconda-client/blob/662968b37fa61018a3e8524dc3ea5084e9aa4037/binstar_client/mixins/package.py#L15-L16 )

That said, not sure what counts as docs for `anaconda-client`. The [GitHub project's README links]( https://github.com/Anaconda-Platform/anaconda-client#quickstart ) to [this page]( https://docs.anaconda.org/free/anacondaorg/ ), which [also shows the CLI]( https://docs.anaconda.org/free/anacondaorg/commandreference#copy ), but not much more

<details>

<summary><code>anaconda copy --help</code></summary>

```bash
usage: anaconda copy [-h] [--to-owner TO_OWNER] [--from-label FROM_LABEL]
                     [--to-label TO_LABEL] [--replace | --update]
                     spec

Copy packages from one account to another

positional arguments:
  spec                  Package - written as user/package/version[/filename]
                        If filename is not given, copy all files in the
                        version

options:
  -h, --help            show this help message and exit
  --to-owner TO_OWNER   User account to copy package to (default: your
                        account)
  --from-label FROM_LABEL
                        Label to copy packages from
  --to-label TO_LABEL   Label to put all packages into
  --replace             Overwrite destination package metadata
  --update              Update missing data in destination package metadata
```

</details>

Reply:
We could use `update` if we prefer to keep the existing metadata

Went with `replace` as I was concerned old metadata would hang around (like dead links, outdated descriptions, etc.)

```suggestion
                    update=True,
```

Reply:
I'm worried about overwriting the actual package contents. I cannot tell from the api client source and docs if these flags are just for metadata or are for the package itself too.

Reply:
That sounds like a reasonable concern

Would be happy to try with `update` first and follow up on `replace` separately

What do you think?

Reply:
yes i'd be happy to ship update now and punt on replace until we do more testing.

Reply:
We may also hear back from anaconda on what these do.

Reply:
Sounds like we want `update` ( https://github.com/conda-forge/conda-forge-webservices/pull/559#issuecomment-1590241850 ). Will make this change